File: MES_SPRING/src/main/java/com/hm/mes_final_260106/dto/TokenDto.java
package com.hm.mes_final_260106.dto;

import lombok.*;
// 로그인 성공 시 응답으로 클라이언트에 전달

@Getter @Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TokenDto {
    private String grantType;
    private String accessToken;
    private String refreshToken;
    private Long accessTokenExpiresIn;
    private Long refreshTokenExpiresIn;
}

File: MES_EQUIPMENT/Models/TokenDto.cs
// 백엔드에서 제외
public class TokenDto
{
    public string GrantType { get; set; } = string.Empty;
    public string AccessToken { get; set; } = string.Empty;
    public string RefreshToken { get; set; } = string.Empty;
    public long AccessTokenExpiresIn { get; set; }
}